packages/manager/modules/pci/src/projects/project/storages/instance-backups/instance-backup/delete/delete.controller.js

```diff
@@ -11,7 +11,15 @@ export default class PciBlockStorageDetailsDeleteController {
   }
 
   $onInit() {
-    this.isLoading = false;
+    this.isLoading = true;
+    return this.PciProjectStorageInstanceBackupService
+      .getAssociatedInstance(this.projectId, this.instanceBackup)
+      .then((installed) => {
```

Comment:
```suggestion
      .then((instance) => {
```
? Looks more logical that way